zimui/src/components/channel/tabs/LivesTab.vue

```diff
@@ -0,0 +1,18 @@
+<script setup lang="ts">
+import { useMainStore } from '@/stores/main'
+import { computed } from 'vue'
+
+import LivesGridTab from './LivesGridTab.vue'
+
+const main = useMainStore()
+const livesAvailable = computed(() => main.channel?.userLivesPlaylist)
+</script>
+
+<template>
+  <div v-if="livesAvailable">
```

Comment:
Do we need this? Lives tab is anyway not shown when there is not short, isn't it?